cmd/oci-image-tool/man/oci-image-tool-unpack.1.md

```diff
@@ -2,7 +2,8 @@
 % OCI Community
 % JULY 2016
 # NAME
-oci-image-tool-unpack \- Unpack an image or image source layout
+oci-image-tool-unpack \- Validates an application/vnd.oci.image.manifest.v1+json and unpacks
+its layered filesystem to `dest`.
```

Comment:
I'd rather have this new stuff under DESCRIPTION [too][1].

[1]: https://github.com/opencontainers/image-spec/pull/241/files#r77212739